app/src/main/java/br/com/lucas/pomodoroapp/database/di/AppModule.kt

```diff
@@ -0,0 +1,33 @@
+package br.com.lucas.pomodoroapp.database.di
```

Comment:
**#suggestion**
 - Move the di package to the same database level:
- app
       |-> database
        |-> di
        | ...